keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -64,22 +53,31 @@ class ResourceActuator(
       return
     }
 
-    log.debug("Checking resource {}", id)
-
-    if (resourceRepository.lastEvent(id) is ResourceActuationPaused) {
-      log.info("Actuation for resource {} resuming", id)
-      publisher.publishEvent(
-        ResourceActuationResumed(
-          resource.apiVersion,
-          resource.kind,
-          id.value,
-          id.application,
-          clock.instant()))
-    }
-
     try {
       val (desired, current) = plugin.resolve(resource)
       val diff = ResourceDiff(desired, current)
+      diffFingerprintRepository.store(id, diff)
```

Comment:
Note: if there's no diff, this stores a hash of "{}". Is that desired behavior? Or, if there's no diff, we could just clear the record of the diff...

Reply:
Undesired behavior. Changing so that if there's no diff, we don't store anything (but the old record remains). I can't think of any problems with this right now since flapping = unhappy status + same diff more than 10 times in a row